tests/queries/0_stateless/02352_grouby_shadows_arg.sql

```diff
@@ -0,0 +1,5 @@
+SET prefer_localhost_replica=0;
+SELECT toString(dummy) as dummy FROM remote('127.{1,1}', 'system.one') GROUP BY dummy;
```

Comment:
Please add `-- { echoOn }`, so reference file will contain query and corresponding result